packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx

```diff
@@ -133,10 +142,7 @@ export class AtomicQuerySummary implements InitializableComponent {
 
     return (
       <div class="text-on-background" part="container">
-        {this.querySummaryState.hasResults && [
-          this.renderHasResults(),
-          this.renderDuration(),
-        ]}
+        {this.querySummaryState.hasResults && this.renderHasResults()}
```

Comment:
Visual glitch I encountered trying to render long queries with an ellipsis (I'll probably open an official PR for this)